backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java

```diff
@@ -57,8 +57,10 @@ public List<Discussion> askLLMQuestion(UUID id, ConversationRequest conversation
                     .build());
         }
 
-        // TODO might needed to process the text
-        String resultFromAssistant = getPrediction(conversationRequest.getText());
+        String prompt = initializeModel();
+        prompt += preprocessPrompt(conversation.get().getDiscussions());
+        prompt += conversationRequest.getText() + " [/INST] ";
```

Comment:
`[/INST]` can we have this as a final static in `llmservice`  ?

and it can be used there as well since this is being used alot